src/oneid/oneid-fe/yarn.lock

```diff
@@ -4701,9 +4712,9 @@ vite@^5.0.0:
     fsevents "~2.3.3"
 
 vite@^6.2.3:
-  version "6.2.3"
-  resolved "https://registry.yarnpkg.com/vite/-/vite-6.2.3.tgz#249e92d32886981ab46bc1f049ac72abc6fa81e2"
-  integrity sha512-IzwM54g4y9JA/xAeBPNaDXiBF8Jsgl3VBQ2YQ/wOY6fyW3xMdSoltIV3Bo59DErdqdE6RxUfv8W69DvUorE4Eg==
+  version "6.2.4"
```

Comment:
@himazawa @FilippoBau it shouldn't update the package in that way. This change should occur on `package.json` file and reflecting here in `yarn.lock`

Reply:
thanks @libremente I think I just noticed the difference between renovate and dependabot which operates in that way